packages/api/src/api/contributions.ts

```diff
@@ -0,0 +1,44 @@
+import { Router } from 'express';
+import { Contribution } from '../entities/Contribution';
+import { User } from '../entities/User';
+import logger from '../logger';
+
+export const contributions = Router();
+
+contributions.get('', async (req, res) => {
+  if (!req.user) {
+    res.sendStatus(401);
+    return;
+  }
+
+  const { userId } = req.query;
+
+  if (!userId) {
```

Comment:
I think `userId` will always be `string | undefined` but if it was a `number 0`, this would be a bug 😬 

Also because these are the same status codes, we might as well bundle them. Annnnd because we really don't care about differentiating between `undefined` or `not a number`, we might as well bundle those too and save a test!

Also, what happens if someone sends in an ID of `-1`? That's not `NaN`, is it? 😉 
```suggestion
  if (Number.isNaN(Number(userId)) || Number(userId) < 0) {
```